---
categories: ["cancer genomics", "epigenetics", "intro"]
date: May 17, 2022
---


# Interplay between the cancer genome and epigenome {.unnumbered}

[Cell](https://www.sciencedirect.com/science/article/pii/S0092867413002961?via%3Dihub)

March 28, 2013

> Shen H, Laird PW. Interplay between the cancer genome and epigenome. Cell.
> 2013 Mar 28;153(1):38-55. doi: 10.1016/j.cell.2013.03.008. PMID: [23540689](https://pubmed.ncbi.nlm.nih.gov/23540689/); PMCID: PMC3648790.

## DNA methylation mechanism

While genetic mechanisms of oncogenesis are more easily seen and understood,
epigenetic mechanisms can be powerful ones that can be transferred through
cell divisions. Mutations in epigenetic regulators are important lead to major
changes in cancer cell epigenomes. Genetic mechanisms can inactivate
tumor-supressor genes or work with epigenetic mechanisms to make mutations and
epigenetic dsyregulation more likely.


####  CpG cytosine-5 methylation

This is catalysed by DNMT3A and DNMT3B and maintained by DNAMT3B. Most of the
mammalian genome is not methylated. The sparse methylated regions contain
heavily methylated CpG dinucleotides and unmethylated CpG islands that are
protected from DNA methylation by guanine-cytosine strand asymmetry and R loop
formation. There may also be active demethylation by TET proteins. CpG islands
are present on the start sites of half of human genes - genes that are actively
expressed or about to be.

Methyl-CpG binding domains (MBD) or C2H2 zinc fingers recognize methylation DNA.
MBDs and Kaiso (zinc finger) are involved in DNA methylation-mediated
transcriptional repression of tumor suppressor genes with promoter DNA
methylation.

#### Histone variants and modifications

Histone modifications are carried out by histone methyltransferases 
demethylases, acetyltransferases and deacetylases, and writers and erasers of
phosphorylation and other modifications. They act in complexes like the Polycomb
complec and counterbalance each other to regulate developmental genes, some of
which are implicated in cancer. The Polycomb repressive complex 2 (PRC2) allows
PRC1 to dock. PRC1 has a RING1B enzymatic core that monoubiquitinylates histone
H2A at K119 and blocks RNA polymerase II elongation. The H3K4 methylation mark
made by the Trithorax group complex counters Polycomb function. PRC2 targets
embryonic stem cell transcription factors for differentiation and development
regulators with the activating H3K4me3 and the repressive H3K27me3 marks. The
Trithorax demethylase KDM6A can remove this repressive mark allowing for gene
transcription for differentiation. Genes not needed for this process lose the
activating mark and gain the repressive one.

Histone variants are found in different cell types and regions. Some like H2A.Z
are found in regions requiring frequent changes and wide prevalence indicates
prevalent chromatin exchange in embryonic stem cells.

#### Nucleosome positioning and remodelling

Nucleosome position can influence transcriptional activation and repression.
There are chromatin remodelling complexes (SWI/SNF, chromodomain and
helicase-like domain(CHD), ISWI and INO80) that can change nucleosome positions


Other epigenetic changes include:

- Hydroxylation
- Formylation
- Carboxylation

These changes are made by

- initiators like lncRNAs,
- writers establish marks, guided by sequence context, existing marks
  and bound proteins, ncRNAs, or nuclear architecture. Mutations to writers
  like *DNMT3A* are often found in cancers and  
- readers that interpret them,
- erasers like TET that remove them,
- remodelers that reposition nucleosomes and 
- Insulators 

    Insulators form boundaries between epigenetic domains and include the
    CCCTC-binding factor CTCF and CTCFL/BORIS. CTCF prevents enhancer
    interactions with unintended promoters. It also separates active and
    repressive chromatin domains.

#### Epigenetic state maintainance

The epigenome is copied between cell divisions by DNMT1 and H3K9
methyltransferase which are loaded into the replication fork and copy the marks.
The Trithorax and Polycomb complexes are recruited during replication and
distributed between the daughter cells to restore marks.

## Epigenetic disruption in cancer

Studies have found contradictory changes in DNA methylation  in cancer. Some
find hypomethylation while others find hypermethylation of CpG islands. We know
epigenetic changes can be causal in cancer since many tumor-supressor genes have
been silenced by promoter CpG islands hypermethylation independent of mutational
or structural inactivation of the gene. Further mouse models need epigenetic
readers and writers for tumor development. Some methylation changes are
essential for cancer cell survival.

#### Disruption of differentiation and development

Polycomb repressors mark genes differentiation and development regulator genes
depending on that cells trajectory. These genes tend to be hypermethylated in
their CpG islands during proliferation, ageing and malignant
transformation. Cancer cells also show silencing of differentiation genes. This
suggests a model where Polycomb target genes in stem cells acquire permanent
silencing over time and lose their ability to differentiate, but are still able
to proliferate. This would mean that it is an epigenetic modification rather
than a genetic one that leads to malignant transformation.

#### CpG island methylator phenotypes 

The CpG island methylator phenotype shows a high frequency of CpG island
hypermethylation. There are distinct epigenetic subtypes for some cancers like
CRC and glioblastoma, while others like ovarian cancer do not show such distinct
subtypes.

#### Epigenetic influences on genomic integrity

Epigenetic mechanisms can affect the rates of lesion formation and repair.
Methylated and unmethylated cytosine can both be deaminated. Unmethylated C
forms uracil and 5mC forms thymine. Since uracil is not part of normal DNA is
it more likely to be found and repaired than a change to thymine which leads to
a 10-fold higher rate of C-to-T mutations in CpG dinucleotides that mostly
contain MeC than other SNVs in the genome. The deamination of 5mC before
replication leads to a normal T:A base that is not a repair candidate. About a
quarter of *TP53* mutations in cancer result from this epigenetic mark.

#### Regional effects of chromatic organization

Megabase regions of repressive chromatin (H3K9me3) are correlated with cancer
SNVs. Genes resistant to cancer-associated hypermethylation tend to have
retrotransposons near their transcription start sites compared to
methylation-prone genes. Mouse models of DNMT deficiency tend to have
chromosomal instability.

#### Epigenetic influences on DNA repair

Removal of DNA methyltransferases can lead to increased microsatellite
instability (MSI), increased telomere lengths and telomeric recombination. The
epigenetic silencing of DNA repair genes like *MLH1* can lead to an increase in
mutation rates and genomic instability in cancer cells.

Mutations of epigenetic regulator genes like writers and erasers can be
mechanisms of oncogenesis. *DNMT3A* and *TET2* mutations have been found in
cancers. 

Histone gene mutations can also be involved in cancer as they affect histone
positioning and telomeres. Some like the K27M mutation mimic a Polycomb
repressive mark. Histone methylation writer mutations like in *MLL* are found in
leukemias. This may work by the inappropriate recruitment of epigenetic factors
to MLL targets. Some writers like *EZH2* have both gain-of-function and
loss-of-function mutations in cancer. This may be part of a control system with
different functions in different environments that are dysregulated when
mutations occur in these genes.

Chromatin remodelling involves remodelling the structure and
positioning of nucleosomes to enable access. Chromatin remodeler mutations are
implicated in a number of different cancers. SWI/SNF complexes are remodelers
with mutations in rhabdoid tumors.
