---
date: March 18, 2021
---

# A multidimensional systems biology analysis of cellular senescence in aging and disease {.unnumbered}

[Genome Biology](https://doi.org/10.1186/s13059-020-01990-9)

April 07, 2020

> Avelar RA, Ortega JG, Tacutu R, Tyler EJ, Bennett D, Binetti P, Budovsky A,
> Chatsirisupachai K, Johnson E, Murray A, Shields S, Tejada-Martinez D,
> Thornton D, Fraifeld VE, Bishop CL, de Magalhães JP. A multidimensional
> systems biology analysis of cellular senescence in aging and disease. Genome
> Biol. 2020 Apr 7;21(1):91. doi: 10.1186/s13059-020-01990-9. PMID:
> [32264951](https://pubmed.ncbi.nlm.nih.gov/32264951); PMCID: PMC7333371.

## Introduction

Hayflick and Moorhead demonstrated replicative senescence in fibroblasts that
reached a stable proliferative growth arrest. The cells remained metabolically
active and had a distinct vacuolated morphology.

Replicative senescence is driven by telomere attrition that exposes the
chromosome end and triggers the DNA damage response. Other types of senescence
occur with exposure to stressors like oncogenic factors, oxidative stress or
radiation. It is a pleiotropic phenomenon vital to embryogenesis, tumour
suppression, and wound healing.

The persistence of senescence cells without clearance by the immune system can
lead to a chronic state of senescence and inflammation through the
senescence-associated secretory phenotype - a state that can spread senescence
to neighbouring healthy cells.

It can be hard to determine what cellular senescence is since there is no one
marker to identify it. β-galactosidase activity and p16 expression are usually
used to identify senescent cells, but these markers are not unique or always
found in senescent cells. The CellAge database uses existing data on human
ageing to provide data on senescence.

## Results

CellAge has 279 senescence genes in different senescence pathways. They ran a
meta-analysis to get a molecular signature of replicative senescence and found
526 overexpressed and 734 underexpressed genes. 44 of the 279 CellAge genes were
found in cellular senescence signatures, a significant overlap.

### CellAge gene functions

Functional enrichment analyses showed that CellAge genes are enriched with
several clusters associated with Protein Kinase Activity, transcription
regulation DNA-binding, DNA damage repair, and cell cycle regulation in cancer.
Genes inducing senescence were more associated with promoting transcription and
genes inhibiting senescence were associated with transcription repression.

Clustering related processes resulted in 298 categories that were significantly
enriched.

### Evolutionary conservation of CellAge genes in model organisms

There were significantly more orthologues for human CellAge genes in mice, rats,
and monkeys than for other genes. Senescence inducers were more conserved than
inhibitors.

22 orthogroups were conserved in the 24 mammals studied.

### CellAge vs human orthologues of longevity-associated model organism genes

Senescence inducers overlapped with anti-longevity genes and not with
pro-longevity genes and vice-versa with senescence inhibitors. They did however
find an overrepresentation of senescence inhibitors in the anti-longevity genes
set. There is a significant association between cellular senescence and the
ageing process and there may be an inverse relationship between senescence and
longevity for some pathways.

### CellAge genes differentially expressed with age

Genes overexpressed with age overlapped with the CellAge genes and genes
underexpressed with age did not show any significant overlap. Overexpressed
signatures of ageing overlapped with overexpressed signatures of replicative
senescence, but not underexpressed signatures of ageing.

Underexpressed signatures of replicative senescence did not overlap with
overexpressed or underexpressed ageing signatures.

91 inducers and 72 inhibitors overlapped with ageing signatures in fibroblasts.

Functional enrichment analysis showed that 71 GO terms were enriched for the
overlap between CellAge senescence inducers and age upregulated genes.

### Tissue specific senescence gene expression and differential expression of

They asked if CellAge genes and differentially expressed signatures of
senescence were tissue-specific. CellAge genes were generally expressed across
tissue types. Only 10% of CellAge genes and 11% of senescence signatures had
tissue-specific expression.

Overexpressed senescence signatures were across tissue types with age. 64% of
protein-coding genes did not significantly change expression with age in any
tissues. The number of CellAge senescence inducers overexpressed with age was
higher than the genome average.

Overexpressed signatures of senescence were significantly differentially
expressed with age compared to all protein-coding genes. The number of
underexpressed signatures of senescence was underexpressed more than expected by
chance.

The expression of most of the senescence genes did not change with age, but a
significant number tend to be differentially expressed with age across multiple
tissues in humans.

### Senescence and longevity genes associate with ageing related-diseases

Using data on 769 ageing related diseases, they found that all were
significantly associated with CellAge genes.

### Are CS genes associated with cancer genes?

There was a significant overlap between CellAge genes and cancer, with both
oncogenes and tumour suppression genes.

Senescence inducers tended to be tumour suppressors, while inhibitors were
oncogenes which is consistent with the view that senescence is a
tumour-suppressing mechanism.

### Network analysis

RNA-Seq Unweighted Co-expression network:

There were a number of clusters found and one was related to immunity.
