April 19, 2021

# Transcriptome signature of cellular senescence

June 28, 2019

> Casella G, Munk R, Kim KM, Piao Y, De S, Abdelmohsen K, Gorospe M.
> Transcriptome signature of cellular senescence. Nucleic Acids Res. 2019 Dec
> 2;47(21):11476. doi: 10.1093/nar/gkz879. Erratum for: Nucleic Acids Res. 2019
> Aug 22;47(14):7294-7305. PMID:
> [31612919](https://pubmed.ncbi.nlm.nih.gov/31612919); PMCID: PMC6868356.

Cellular senescence has a number of characteristics including DNA damage,
induction of the p53/CDKN1A (p21) and CDKN2A (p16)/pRB pathways,
senescence-associated Î²-galactosidase activity, cytoskeletal alterations, the
formation of senescence-associated heterochromatin foci, and the SASP. However,
these markers of senescence are not universal and are not a good way to ensure
that a cell is senescent.

This study used universally expressed transcripts from different senescent cell
models and, using RNA-seq, found RNAs that could be used to identify senescent
cells.

## Results

The main differences in RNA expression depended on the cell type of origin, and
not on the senescence induction method which supports the idea that senescence
is not a single phenotypic state.

The RNA-seq analysis revealed 50 elevated and 18 reduced transcripts between the
senescent and non-senescent cells, some of which were consistent between all
senescence models and were more uniform than the p21 and p16 markers.

Examining the 68 transcripts shared between the senescent types, a PCA showed
that senescence state gave the clearest separation between the cell types. An ML
models using LASSO showed that 5 transcripts, *SLCO2B1*, *CLSTN2* mRNAs and
*LINCO2154* and *PURPL* lncRNAs, were the minimum needed to distinguish
senescent cells. *PURPL* was one of the most highly and consistently elevated
transcripts and is controlled by the transcription factor p53, which is elevated
in senescent cells. It may contribute to the survival of senescent cells as it
does in tumors.
