---
categories: ["chromatin", "enhancers", "gene expression", "gene regulatory networks"]
date: May 22, 2013
date-modified: May 14, 2025
doi: 10.1093/nar/gkt374
---

# Enhancer networks revealed by correlated DNAse hypersensitivity states of enhancers

[Nucleic Acids Research](http://academic.oup.com/nar/article-pdf/41/14/6828/18782613/gkt374.pdf)

> Malin J, Aniba MR, Hannenhalli S. Enhancer networks revealed by correlated
> DNAse hypersensitivity states of enhancers. Nucleic Acids Res. 2013
> Aug;41(14):6828-38. doi: 10.1093/nar/gkt374. Epub 2013 May 21. PMID:
> [23700312](https://pubmed.ncbi.nlm.nih.gov/23700312); PMCID: PMC3737527.

This study aimed to understand the higher order functional organization of
enhancers using P300-bound regions as enhancer candidates. Using DNAse
hypersensitivity, which shows uncondensed regions and marks enhancers, they
found correlated activity between enhancers that decreases with increasing
genomic distance. In their methods, they controlled for autocorrelation and
transitive correlations. They also created background enhancer pairs that were
known to have no correlations as a control while foreground enhancer pairs
where correlated enhancers. They show that genes are organized into "co-expressed
modules of correlated activity across cell types".

Correlated enhancers were spatially proximal to each other and shared common
transcription factor binding motifs, with shared motifs in enhancer pairs
predicting correlated activity with 73% accuracy. Co-occuring motifs were more
prevalent for correlated enhancers that were farther apart than for those in
close proximity. Genes near correlated enhances also have correlated expression
and shared function as did the targets of correlated enhancer clusters. Of the
correlated enhancers, strong and weak enhancers tended to correlate more among
themselves than with each other, with weak enhancers having more correlations
than strong enhancers.

Since enhancers are involved in cell-type-specific gene expression, if an
enhancer cluster's target genes are cell-type-specific, those enhancers should
have high DNAse hypersensitivity sites. They tested whether there is
cell-specificity among the targets of correlated enhancers and found "concordance
between enhancer clusters and targeted gene clusters in their tissue-specific
activity".
